src/Component/Person/Person.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import './Person.scss';
+
```

Comment:
I think we should use default proptype here as we have some properties absent for some people.

Reply:
I check with Paul, it is optional on the task.

Reply:
Right!